### PR TITLE
Added lumen

### PR DESCRIPTION
**Pull request by Hoxbro**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by philippjfr** [comment]
I confirm I'd like to be added as maintainer of this recipe.

File: recipes/lumen/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Copyright (c) 2018-2020, HoloViz team (holoviz.org).
```
Comment (dopplershift) [inline recipes/lumen/LICENSE.txt:1]: The PyPI tarball already has `LICENSE` with the same content. Is there any need to include this here?

File: recipes/lumen/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  host:
+    - python >=3.7
+    - pip
+    - param >=1.7.0
+    - pyct >=0.4.4
+    - setuptools >=30.3.0
+  run:
+    - python >=3.7
+    - panel >=0.11
+    - hvplot
```
Comment (dopplershift) [inline recipes/lumen/meta.yaml:30]: Should this also have the `pandas` and `holoviews` dependencies listed upstream?

File: recipes/lumen/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+about:
+  home: https://github.com/holoviz/lumen
+  summary: 'Illuminate your data. A monitoring solution built on Panel'
+  description: |
+    The Lumen project provides a framework for visual analytics, which allows users to
+    build data-driven dashboards from a simple yaml specification. The power of
+    Lumen comes from the ability to leverage the powerful data intake, data processing
+    and data visualization libraries available in the PyData ecosystem.
+  license: BSD-3-Clause
+  license_file: LICENSE.txt
```
Comment (dopplershift) [inline recipes/lumen/meta.yaml:50]:
```suggestion
  license_file: LICENSE
```
?

**Review by dopplershift — CHANGES_REQUESTED** [review]
A few minor comments, then this should be good to go.

File: recipes/lumen/LICENSE.txt (hunk 1 above)
Comment (Hoxbro) [reply recipes/lumen/LICENSE.txt]: No.

File: recipes/lumen/meta.yaml (hunk 2 above)
Comment (Hoxbro) [reply recipes/lumen/meta.yaml:30]: I have added them. 

I was not sure how much I should add under run, the bare minimum or all of the packages needed.

Comment (dopplershift) [reply recipes/lumen/meta.yaml:30]: Bare minimum is fine, but then I have to ask: why are they listed under `install_requires` in `setup.py`? :wink:

**Review by dopplershift — APPROVED** [review]

File: recipes/lumen/meta.yaml (hunk 2 above)
Comment (Hoxbro) [reply recipes/lumen/meta.yaml:30]: That is such a good question, which I can't answer :upside_down_face: 

Thank you for your guidance and help.

**Pull request merged**